.run/spring-DiscoverableClientSampleApplication.run.xml

```diff
@@ -1,5 +1,35 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration name="spring-DiscoverableClientSampleApplication" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+  <configuration default="false" name="spring-DiscoverableClientSampleApplication" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
```

Comment:
Does this need to be part of the PR?

Reply:
I have no idea. It came from your commit. Feel free to remove it.

Reply:
hahaha ok